config.py

```diff
@@ -14,6 +14,7 @@
 
 DEFAULT_BROKER_IP = "192.168.100.100"
 DEFAULT_BIKE = "V2"
+DEFAULT_VIEWPORT_SIZE = [1024, 600]
```

Comment:
Thoughts on having this in a defaults file? Not sure if this is used throughout the codebase or if they're only used in this file.
Seems like DEFAULT_BROKER_IP would be something that we want to be shared in the code base unless there is a function for that alraedy.

Reply:
This kind of acts as the defaults file already, since raspicam always grabs configuration using `read_configs()`.